lib/utils/ConfigParser.js

```diff
@@ -272,6 +256,44 @@ class ConfigParser {
         return this._capabilities
     }
 
+    /**
+     * sets config attribute with file paths from filtering
+     * options from cli argument
+     *
+     * @param  {String} filterObj  list of files in a string from
+     * cli argument
+     * @return {Boolean} exclude  true or false
+     */
+    setFilePathToFilterOptions (filterObj, exclude = false) {
+        const FILTER_TYPE = exclude ? 'exclude' : 'specs'
+
+        const objs = new Set()
+        const objList = filterObj.split(/,/g)
+        const objsList = ConfigParser.getFilePaths(this._config[FILTER_TYPE])
```

Comment:
Instead of having to set `exclude=true/false` let's pass in `ConfigParser.getFilePaths(this._config[FILTER_TYPE])` as second parameter which makes the function more open.

Reply:
ok, I was a bit bothered with all the exclude = true thing... I am happier with this approach.